_docker/drupal/README.md

```diff
@@ -35,6 +35,19 @@ The `composer create-project` command passes ownership of all files to the
 project that is created. You should create a new git repository, and commit 
 all files not excluded by the .gitignore file.
 
+## Restoring content to Drupal
+
+If something goes wrong and you need to restore the drupal environment from a backup the steps are pretty easy:
+
+1. Obtain the backup. This will include an sql file an a tar.gz file.
+2. Extract the tar.gz file into `_docker/drupal/web`.
+3. Execute `control.rb -e drupal-dev --run-the-stack`.
```

Comment:
@LightGuard Perhaps make it clear that the person following these instructions needs to specify the environment (and also container names) correctly. For example, these instructions won't work if I try them in production environment as the container names will be different